### PR TITLE
Fix device Type V3

### PR DESCRIPTION
**Pull request by FabianKielkopf**
Added support for new Devices (Gen 3 Nuki Startlocks).

### CODE REVIEW

**Comment by c0ffee** [comment]
+1

File: pynuki/bridge.py [hunk 1]
```diff
@@ -248,38 +248,38 @@ def locks(self):
     def openers(self):
         return self._get_devices(device_type=const.DEVICE_TYPE_OPENER)
 
-    def lock(self, nuki_id, block=False):
+    def lock(self, nuki_id, device_type, block=False):
```
Comment (pschmitt) [inline pynuki/bridge.py:251]:
```suggestion
    def lock(self, nuki_id, device_type=const.DEVICE_TYPE_LOCK, block=False):
```

File: pynuki/bridge.py [hunk 2]
```diff
@@ -248,38 +248,38 @@ def locks(self):
...
         return self.lock_action(
             nuki_id,
             action=const.ACTION_LOCK_LOCK,
-            device_type=const.DEVICE_TYPE_LOCK,
+            device_type=device_type,
             block=block,
         )
 
-    def unlock(self, nuki_id, block=False):
+    def unlock(self, nuki_id, device_type, block=False):
```
Comment (pschmitt) [inline pynuki/bridge.py:259]:
```suggestion
    def unlock(self, nuki_id, device_type=const.DEVICE_TYPE_LOCK, block=False):
```

File: pynuki/bridge.py [hunk 3]
```diff
@@ -248,38 +248,38 @@ def locks(self):
...
         return self.lock_action(
             nuki_id,
             action=const.ACTION_LOCK_UNLOCK,
-            device_type=const.DEVICE_TYPE_LOCK,
+            device_type=device_type, 
             block=block,
         )
 
-    def lock_n_go(self, nuki_id, unlatch=False, block=False):
+    def lock_n_go(self, nuki_id, device_type, unlatch=False, block=False):
```
Comment (pschmitt) [inline pynuki/bridge.py:267]:
```suggestion
    def lock_n_go(self, nuki_id, device_type=const.DEVICE_TYPE_LOCK, unlatch=False, block=False):
```

File: pynuki/bridge.py [hunk 4]
```diff
@@ -248,38 +248,38 @@ def locks(self):
...
         return self.lock_action(
             nuki_id,
             action=action,
-            device_type=const.DEVICE_TYPE_LOCK,
+            device_type=device_type, 
             block=block,
         )
 
-    def unlatch(self, nuki_id, block=False):
+    def unlatch(self, nuki_id, device_type, block=False):
```
Comment (pschmitt) [inline pynuki/bridge.py:278]:
```suggestion
    def unlatch(self, nuki_id, device_type=const.DEVICE_TYPE_LOCK, block=False):
```

**Review by pschmitt — CHANGES_REQUESTED** [review]
I'd like to have a default value set for `device_type` so that this is less of a breaking change.
Other than that: LGTM, thanks!

**Comment by mi1os** [comment]
hey guys, can we please merge this? can I help with it somehow?

**Comment by pvizeli** [comment]
Yeah, would be nice to having it

**Pull request merged**